small_birthday_gift_A&V.ipynb

```diff
@@ -0,0 +1,301 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## small gift for birthday\n",
+    "\n",
+    "dear friends!\n",
+    "\n",
+    "recently bostrom had his first birthday!\n",
+    "\n",
+    "since the main gift cannot be unlocked until we reach the goal, the cybercongress decided to make a small gift\n",
```

Comment:
changes from first review to reflect here

Reply:
done